notify/mail.sh

```diff
@@ -55,8 +62,9 @@ mail_send() {
 
   contenttype="text/plain; charset=utf-8"
   subject="=?UTF-8?B?$(echo "$_subject" | _base64)?="
-  result=$({ echo "$_MAIL_BODY" | eval "$_MAIL_CMD"; } 2>&1)
+  result=$({ _mail_body | _mail_cmnd; } 2>&1)
```

Comment:
This does not work for me. Had to change it to suggested to make it work.
```suggestion
  result=$({ echo "$(_mail_body)" | eval $(_mail_cmnd); } 2>&1)
```